metricbeat/module/openmetrics/_meta/fields.yml

```
@@ -7,18 +7,45 @@
   fields:
     - name: openmetrics
       type: group
+      #      release: beta/ga
       description: >
         `openmetrics` contains metrics from endpoints that are following Openmetrics format.
       fields:
         # Order is important here, labels will match first, the rest are double
+        - name: help
+          type: keyword
+          description: >
+            Brief description of the MetricFamily
+        - name: type
+          type: keyword
+          description: >
+            metric type
+        - name: unit
+          type: keyword
+          description: >
+            metric unit
+        - name: created
+          type: keyword
+          description: >
+            metric creation time in seconds
```

```suggestion
            Metric creation time in seconds
```
Wonder if this info is useful at all...:thinking:

---

changed to Metric